tests/component_test.py

```diff
@@ -266,8 +266,9 @@ def run_test_relay(release_dir: pathlib.Path,
         client_rel = tests.relay.RemoteCaller(url_rel)
 
         # relay a message to the mock MailGun server
-        message = tests.relay.Message(subject="a message from your friend", content="dear friend, I hope all is good.")
-        resp = client_rel.put_message(descriptor=desc, token=token, message=message)
+        message = tests.relay.Message(subject="a message from your friend",
+                                                 content="dear friend, I hope all is good.")
```

Comment:
Can you please add yapf, pydocstyle and other python pre-commit checks? (I suppose you need to include them in the requirements.txt file with version constraints?)

IMO, it would make most sense to move requirements.txt to test/ directory and have a separate test/precommit.py there that expects a virtual environment with the given requirements.

You can then include both precommits (the one that checks the production Go code and the one that checks the Python test code) in .travis.yml for continuous integration.

Please consult .travis.yml in our other projects.